notebooks/random_forest_clean.ipynb

```diff
@@ -0,0 +1,1507 @@
+{
```

Comment:
<p>If you like the below visualization better, delete these two scatter plots</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>oh nice! I like the violin plot, let's keep that one</p>